### PR TITLE
JA: resync notebooks in site/ja/tutorials/quickstart with en master

### PR DESCRIPTION
**Pull request by masa-ita**

### CODE REVIEW

File: site/ja/tutorials/quickstart/beginner.ipynb [hunk 1]
```diff
@@ -1,235 +1,256 @@
...
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "0trJmd6DjqBZ"
+   },
+   "outputs": [],
+   "source": [
+    "from __future__ import absolute_import, division, print_function, unicode_literals\n",
+    "\n",
+    "# Install TensorFlow\n",
```
Comment (AseiSugiyama) [inline site/ja/tutorials/quickstart/beginner.ipynb]: This comment is left without translation. Could you translate it?

File: site/ja/tutorials/quickstart/beginner.ipynb [hunk 2]
```diff
@@ -1,235 +1,256 @@
...
+    "try:\n",
+    "  # %tensorflow_version only exists in Colab.\n",
```
Comment (AseiSugiyama) [inline site/ja/tutorials/quickstart/beginner.ipynb]: Same as above, this comment is left without translation. Could you translate it?

File: site/ja/tutorials/quickstart/advanced.ipynb [hunk 3]
```diff
@@ -1,387 +1,418 @@
...
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "0trJmd6DjqBZ"
+   },
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "  # %tensorflow_version は Colab にだけ存在する\n",
```
[the remark this answers is not in the document]
Comment (masa-ita) [reply site/ja/tutorials/quickstart/advanced.ipynb]: Thank you for your suggestion.

File: site/ja/tutorials/quickstart/beginner.ipynb (hunk 2 above)
Comment (masa-ita) [reply site/ja/tutorials/quickstart/beginner.ipynb]: Thanks.  Translated.

File: site/ja/tutorials/quickstart/beginner.ipynb (hunk 1 above)
Comment (masa-ita) [reply site/ja/tutorials/quickstart/beginner.ipynb]: Thanks.

**Comment by AseiSugiyama** [comment]
LGTM!

**Pull request merged**